gi_notifier/runner.py

```diff
@@ -0,0 +1,32 @@
+from asyncio import Event
+
+from apscheduler.schedulers.asyncio import AsyncIOScheduler
+from pytz import timezone
+from telegram.ext import ApplicationBuilder, ChatMemberHandler, CommandHandler
+
+from . import config
+from .handler.chat_handler import handle_bot_added
+from .handler.command_handler import handel_resinplan, handle_start
+from .scheduler import scheduled_task
+
+
+async def runner():
+    application = ApplicationBuilder().token(config.BOTTOKEN).build()
+    application.add_handler(CommandHandler("start", handle_start))
+    application.add_handler(ChatMemberHandler(handle_bot_added))
+    application.add_handler(CommandHandler("ResinPlan", handel_resinplan))
```

Comment:
Like I have mentioned before, reduce it to `/resin` and keep the script open for shorthands like `/r` for convenience purposes.